public/assets/images/bisecthosting.svg

```diff
@@ -0,0 +1 @@
 4.19 0 0 0 70.8 0c-2.32-.13-4 2-4.11 4.1a4.08 4.08 0 0 0 3.55 4v9.39h-5.73a2.92 2.92 0 0 0-2.74-2.21A2.83 2.83 0 0 0 59 18.11a2.8 2.8 0 0 0 2.78 2.77 2.71 2.71 0 0 0 2.69-2.21h5.78v18.42a.56.56 0 0 0 1.12 0V8.2a4.24 4.24 0 0 0 3.53-4.09zM61.77 19.76c-2.13.11-2.12-3.19 0-3.31a1.66 1.66 0 0 1 0 3.31zm6-15.65c-.21-3.84 5.75-3.82 6 0a3 3 0 0 1-5.95 0zM79.06 119.77V90.82a.56.56 0 0 0-1.12 0v18.42h-5.73a2.9 2.9 0 0 0-2.74-2.24 2.78 2.78 0 1 0 2.7 3.33h5.77v9.35a4.24 4.24 0 0 0-3.54 4.09 4.14 4.14 0 0 0 4.1 4.1c2.32.13 4-2 4.11-4.1a4.16 4.16 0 0 0-3.55-4zm-9.59-8.32a1.65 1.65 0 0 1 0-3.3c2.13-.15 2.12 3.19 0 3.3zm6 12.35c-.21-3.85 5.76-3.83 6 0a3 3 0 0 1-5.95 0z"/><path class="cls-1" d="M102.07 89.9a3.56 3.56 0 0 0-3.43 2.94h-5a.57.57 0 0 0 0 1.13h5a3.62 3.62 0 0 0 3.47 2.94 3.51 3.51 0 1 0 0-7zm0 5.89c-3.07.16-3-4.6 0-4.77a2.39 2.39 0 0 1 0 4.77z"/><path class="cls-1" d="M92.85 115.37v-29a.56.56 0 0 0-1.12 0v18.43H86a2.93 2.93 0 0 0-2.74-2.22A2.78 2.78 0 1 0 86 106h5.78v9.35a4.24 4.24 0 0 0-3.54 4.08 4.11 4.11 0 1 0 4.66-4zm-9.59-8.32a1.65 1.65 0 0 1 0-3.3c2.13-.12 2.11 3.19 0 3.3zm6 12.34c-.21-3.84 5.76-3.82 6 0a3 3 0 0 1-5.95 0zM40.65 111.51v-19.2a.57.57 0 0 0-1.13 0v12.82h-6.93a3.2 3.2 0 0 0-3-2.51 3.08 3.08 0 1 0 3 3.64h7v5.19a4.24 4.24 0 0 0-3.59 4.09c-.13 2.32 2 4 4.11 4.11a4.09 4.09 0 0 0 .56-8.14zm-3.55 4c-.2-3.85 5.76-3.83 6 0a3 3 0 0 1-6 .03zM22.79 82.33v-4.48h5.15a.57.57 0 0 0 0-1.13H14.48a3.22 3.22 0 0 0-3-2.52 3.08 3.08 0 1 0 3 3.65h7.22v4.43a3.65 3.65 0 0 0-2.94 3.48 3.51 3.51 0 1 0 4.07-3.43zm-2.95 3.43c-.16-3.08 4.6-3.06 4.77 0a2.39 2.39 0 0 1-4.77 0zM58.78 106.24v-29a.56.56 0 0 0-1.12 0v24h-5.73a2.92 2.92 0 0 0-2.74-2.21 2.78 2.78 0 1 0 2.69 3.34h5.78v3.76a4.24 4.24 0 0 0-3.54 4.08c-.13 2.32 2 4 4.1 4.11a4.08 4.08 0 0 0 .56-8.13zm-9.59-2.73a1.66 1.66 0 0 1 0-3.31c2.13-.11 2.12 3.19 0 3.31zm6.05 6.75c-.21-3.84 5.76-3.82 6 0a3 3 0 0 1-6 0z"/><path class="cls-1" d="M91.41 28.49H30.46A4.44 4.44 0 0 0 26 32.92v60.94a4.44 4.44 0 0 0 4.42 4.43h61a4.44 4.44 0 0 0 4.42-4.43V32.92a4.44 4.44 0 0 0-4.43-4.43z"/><path class="cls-4" d="M61.24 70.76a6.68 6.68 0 0 1-4 6.34 16.3 16.3 0 0 1-7.36 1.5H35.71a2.23 2.23 0 0 1-2.52-2.52V53a2.23 2.23 0 0 1 2.52-2.52h13.55a25.66 25.66 0 0 1 5.27.41 9.55 9.55 0 0 1 4.82 2.63A6.1 6.1 0 0 1 61.24 58a18.7 18.7 0 0 1-.17 3.35 10.09 10.09 0 0 1-1.65 3.12l-.09.11a6.61 6.61 0 0 1 1.72 2.91 17.23 17.23 0 0 1 .19 3.27zm-6.45-1.56c0-.89-.7-1.43-2.06-1.63a28.65 28.65 0 0 0-3.1-.09h-10V73h9.67a21.49 21.49 0 0 0 3.17-.15c1.38-.25 2.14-.81 2.3-1.68.03-.17.04-.89.02-1.97zm.13-8.93c0-.24-.05-1-.13-2.19s-1.9-2-5.45-2h-9.57v5.8h9.77c1.69 0 2.75 0 3.17-.09 1.49-.18 2.23-.69 2.21-1.52zM88.53 76.08A2.24 2.24 0 0 1 86 78.6a8.62 8.62 0 0 1-2.63-.26 2.16 2.16 0 0 1-1.26-2.26v-8.6H70v8.6a2.23 2.23 0 0 1-2.52 2.52 9.43 9.43 0 0 1-2.67-.24 2.17 2.17 0 0 1-1.26-2.28V53a2.16 2.16 0 0 1 1.28-2.28 9 9 0 0 1 2.65-.24A2.23 2.23 0 0 1 70 53v8.9h12.1V53a2.16 2.16 0 0 1 1.28-2.26 8.37 8.37 0 0 1 2.62-.22A2.24 2.24 0 0 1 88.53 53z"/></g></g></svg>
```

Comment:
Let's have ads under an ads directory to prevent clutter, matching the same as we do for sponsors on home